pkg/api/annotations.go

```diff
@@ -154,7 +150,7 @@ func (hs *HTTPServer) PostAnnotation(c *models.ReqContext) response.Response {
 		Tags:        cmd.Tags,
 	}
 
-	if err := repo.Save(&item); err != nil {
+	if err := hs.annotationsRepo.Save(context.Background(), &item); err != nil {
```

Comment:
I'm not sure, but should these use the request context (c`.Req.Context()`) instead of `context.Background()`?

Reply:
good catch!